src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -762,7 +762,7 @@
 
 helps['aks remove-dev-spaces'] = """
 type: command
-short-summary: Remove Azure Dev Spaces from a managed Kubernetes cluster.
+short-summary: (DEPRECATED) Remove Azure Dev Spaces from a managed Kubernetes cluster.
```

Comment:
pls add deprecate property to command by following https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments